### PR TITLE
[droid] make MenuFlyoutItem.Click working

### PR DESCRIPTION
**Pull request by pkar70**
GitHub Issue (If applicable): #
https://github.com/unoplatform/uno/issues/227

## PR Type

What kind of change does this PR introduce?
- Feature

## What is the current behavior?
MenuFlyoutItem.Click is not fired.

## What is the new behavior?
MenuFlyoutItem.Click is fired.

## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] Tested code with current [supported SDKs] (MonoAndroid90)
- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] [Wasm UI Tests] (not applicable)
- [X] Contains **NO** breaking changes
- [ ] Updated the [Release Notes](https://github.com/unoplatform/uno/tree/master/doc/ReleaseNotes)
- [X] Associated with an issue (GitHub or internal)

## Other information
Seems like some omission in code, and method InvokeClick() as prepared to make Click event works.

### CODE REVIEW

**Comment by pkar70** [comment]
> The modification should be tested with a UI Test. There's already a control that uses MenuBar, 
>it can be enhanced to validated that a menu item is clicked.
I never wrote any automated test before, so writing test would require much learning from my side, and I would not have time for this for at least one week...

**Comment by pkar70** [comment]
Sorry, these two Calendar.... files should not be placed in this PR - I don't know why it was propagated from my clone of Uno to this PR.

File: src/Uno.UI/Controls/CalendarDatePickerValueChangedEventArgs.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using System;
+
+namespace Windows.UI.Xaml.Controls
+{
+	public partial class CalendarDatePickerDateChangedEventArgs
```
Comment (jeromelaban) [inline src/Uno.UI/Controls/CalendarDatePickerValueChangedEventArgs.cs]: This modification should be in another PR as it not related to the MenuFlyout.

**Comment by jeromelaban** [comment]
For UI tests, take a look at this branch https://github.com/unoplatform/uno/commit/a20c2d072706c866f43b925428f0ce7300bbe445 

This uses your changes, and adds some addition updates, and UI Tests.

File: src/Uno.UI/Controls/CalendarDatePickerValueChangedEventArgs.cs (hunk 1 above)
Comment (pkar70) [reply src/Uno.UI/Controls/CalendarDatePickerValueChangedEventArgs.cs]: Yes, I know - I did not know that any changes in my clone of Uno would be automatically replicated to previously created PR. And I don't know how to reverse this (but probably you, Uno Masters, can remove these two changes from PR).

**Comment by jeromelaban** [comment]
@pkar70 did you try to import the tests from the branch I provided you ?

**Comment by pkar70** [comment]
I thought that your test would be added to Uno:master, as it (as you wrote) covers more than only Click event... So we misunderstand each other. Maybe because English is not my primary language...
I don't know how to 'import' automatically; should I re-create changes in 'my branch'?

**Comment by jeromelaban** [comment]
I see. You may want to read a bit more about git, and [cherry-picking in particular](https://www.atlassian.com/git/tutorials/cherry-pick). That will help you take the changes I made into your branch.

**Comment by pkar70** [comment]
But all changes I do not on desktop, where I would be able to use git from command line, but using web: https://github.com/unoplatform/uno/tree/master/src/Uno.UI etc., click on Edit, make changes, click  Commit. It creates branch for PR, it resides on web and not on my disk, and I don't know how to access this branch with commandline commands.
Probably, re-creating changes in my web-branch would be simpler for me than learning GIT on cherry pick level :) The bad thing - your authorship of this changes would be lost.

**Comment by jeromelaban** [comment]
If you want to contribute in a more effective manner, you'll have to learn how to use git to more than what the GitHub web UI is providing you, I'm afraid.

If you're using visual studio, you can take a look at this: https://docs.microsoft.com/en-us/azure/devops/repos/git/gitquickstart?view=azure-devops&tabs=visual-studio. This will explain how to use git in that context, and how to get branches from your remote.
